src/Network/Network/ApplicationGateway/RewriteRule/AzureApplicationGatewayRewriteRuleBase.cs

```diff
@@ -43,7 +53,9 @@ public PSApplicationGatewayRewriteRule NewObject()
             var rewriteRule = new PSApplicationGatewayRewriteRule
             {
                 Name = this.Name,
-                ActionSet = this.ActionSet
+                ActionSet = this.ActionSet,
+                RuleSequence = this.RuleSequence,
+                Condition = this.Condition
```

Comment:
Following Cormac's comment above, to please changes this to `Condition = this.Condition?.ToList()`